### DISCUSSION TITLE
SESSION_DRIVER, CACHE_DRIVER, QUEUE_DRIVER

### DISCUSSION THREAD

**Discussion by stavros-k** [snipe/snipe-it#10869] [Q&A]
I have set the redis host, password and port .

Usually those 3 drivers are also set to redis. But If I set for example `SESSION_DRIVER` or `CACHE_DRIVER` to redis. I get "Please make sure redis extension is installed" from php. 
`QUEUE_DRIVER` set to redis does not show any errors.
<img width="558" alt="image" src="https://user-images.githubusercontent.com/47820033/159982822-c59356f2-4a43-498c-8b0b-3fac4423839a.png">

Does snipe-it support those? Are they supposed to be changed to redis?
Is redis even used by snipe-it?

I looked into docs, but didn't find anything.
Thanks

**Comment by snipe** [comment]
Redis support as a cache driver comes with Laravel, the framework Snipe-IT is built on, but we don't use it or require it specifically, so I can't speak to specific implementation. https://laravel.com/docs/6.x/cache#configuration

**Reply by stavros-k** [reply]
I know about laravel, just wondering if it was any specific use.
Thanks for the answer.
Now I saw that redis is installed as an extension on v6
https://github.com/snipe/snipe-it/commit/57422c9135a9a7c428fdfdd9c8078ad80170cca2
I'll re-test again when it's released (to set the above vars)

**Reply by snipe** [reply]
It's a pretty uncommon use case, tbh - most folks just use the file system, but when v6 is released, I'm anxious to hear how it goes for you with redis :)
